### ISSUE TITLE
random_state with infomax

### ISSUE THREAD

**Issue by teonlamont** [mne-tools/mne-python#2084]
Currently when I run ica with infomax with a random state set, my results are different.

```python
ica = mne.preprocessing.ICA(.9, random_state=42, method='infomax')
ica.fit(raw, decim=4)
```
two separate runs of this code (pay attention to IC 11 and 13):
# **first:**
## ![unknown-4](https://cloud.githubusercontent.com/assets/1578674/7575903/14d41c10-f839-11e4-9182-6416c5f9f8d6.png)

# **second:**
## ![unknown-5](https://cloud.githubusercontent.com/assets/1578674/7575910/2980b218-f839-11e4-96b5-1921839eb787.png)

**Comment by agramfort** [comment]
maybe a good one for @jmontoyam ?

**Comment by teonlamont** [comment]
reproduced using the sample data:

```python
path = mne.datasets.sample.data_path()
raw_file = path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
raw = mne.io.read_raw_fif(raw_file, preload=True)

ica = mne.preprocessing.ICA(.9, method='infomax', random_state=42)
ica.fit(raw, decim=4)
p = ica.plot_components(show=True)

ica2 = mne.preprocessing.ICA(.9, method='infomax', random_state=42)
ica2.fit(raw, decim=4)
p2 = ica2.plot_components(show=True)
```

**Comment by teonlamont** [comment]
@Eric89GXL @dengemann I swear I want a release *tout de suite* but this looks like something that might need to be addressed.

**Comment by Eric89GXL** [comment]
I agree, this is a blocker

**Comment by teonlamont** [comment]
closed for #2096
